packages/core-modules/contracts/storage/ElectionStorage.sol

```diff
@@ -38,6 +38,22 @@ contract ElectionStorage {
          * @dev Position of an address on the nominees Array.
          */
         mapping(address => uint256) nomineeIndexes;
+        /**
+         * @dev Flag to indicate if the election was evaluated (if batched, latest batch was processed).
+         */
+        bool isElectionEvaluated;
+        /**
+         * @dev Used to keep track of the next epoch's winners. Gets erased when an epoch starts and a new council takes effect.
+         */
+        address[] winners;
```

Comment:
Arguable, but `winners` doesn't look right, maybe `nextEpochRepresentatives`?

Reply:
Sort of agree yeah. How are we calling an epoch's NFT holders?
We should call this `nextEpoch<whatever-we-call-those>`

Reply:
done